lang/cs/Org.Apache.REEF.IO.TestClient/Org.Apache.REEF.IO.TestClient.csproj

```diff
@@ -0,0 +1,114 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
REEF.IO is a dll. This one is for yarn testing. After we move tests to NUnit, we won't need this any more. But for now, in order to test on Yarn, we have to write a separated program. I don't want to keep those testing locally as we may need to run it from time to time to test on YARN.
We have a few clients like this, mainly for testing. After moving to NUnit, we should move them into test projects.